packages/generator/src/entity/imports.ts

```diff
@@ -18,13 +17,12 @@ export function entityImportDeclarations(
 ): ImportDeclarationStructure[] {
   return [
     odataImportDeclaration(
-      ['Entity', 'DefaultDeSerializers', 'DeSerializers'],
+      ['Entity', 'DefaultDeSerializers', 'DeSerializers', 'DeserializedType'],
       oDataVersion
     ),
-    odataCommonImportDeclaration(['DeserializedType']),
     ...complexTypeImportDeclarations(entity.properties),
     ...enumTypeImportDeclarations(entity.properties)
-  ];
+  ].sort();
```

Comment:
Does this do, what you expect it to do? I am pretty sure `sort` needs a comparator to be able to handle objects, i.e. `ImportDeclarationStructure`s.